sample-apps/dotnet/Properties/launchSettings.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:37568",
+      "sslPort": 44359
+    }
+  },
+  "profiles": {
+    "myapp": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "applicationUrl": "https://localhost:7148;http://localhost:5083",

```

Comment:
What are these two urls?

Reply:
default ports, they will be overridden by kestrel